qemu/tests/qemu_guest_agent.py

```diff
@@ -102,49 +101,45 @@ def _cleanup_open_session(self):
         except Exception:
             pass
 
-    @error.context_aware
-    def gagent_install(self, params, vm, *args):
+    def gagent_install(self, params, test, vm, *args):
         if args and isinstance(args, tuple):
             gagent_install_cmd = args[0]
         else:
-            raise error.TestError("Missing config 'gagent_install_cmd'")
+            test.error("Missing config 'gagent_install_cmd'")
```

Comment:
if you want to use the new style, it's better to update all

Reply:
I have updated all error.TestError to test.error.